package.json

```diff
@@ -11,12 +13,24 @@
     "url": "https://github.com/filamentgroup/loadJS.git"
   },
   "keywords": [
-    "load"
+    "javascript",
+    "load",
+    "performance"
+  ],
+  "contributors": [
+    "Scott Jehl (http://www.scottjehl.com)",
+    "Filament Group, Inc. <hello@filamentgroup.com> (https://www.filamentgroup.com)"
   ],
```

Comment:
Somewhat annoying that `author` only accepts a string where `contributors` accepts an array…